cluster-logging/eo-rbac.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: prometheus-k8s
+  namespace: openshift-operators-redhat
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - services
+  - endpoints
+  - pods
+  verbs:
+  - get
+  - list
+  - watch
```

Comment:
I'm curious, why do we need to add the Role? That should be managed by OLM: https://github.com/openshift/elasticsearch-operator/blob/release-4.2/manifests/4.2/elasticsearch-operator.v4.2.0.clusterserviceversion.yaml#L106-L119

Reply:
Same as above: I followed the instructions on https://docs.openshift.com/container-platform/4.2/logging/cluster-logging-deploying.html

Reply:
you're right, this should not be needed. I'll verify that

Reply:
removed in 4123686

Reply:
ok, I didn't read the instructions - I didn't come across to them :-) 
they maybe create another SA we should grand the permissions to, but it would be really weird as everything should be managed either by OLM or by the operator itself.